packages/safe-tools-client/app/helpers/payment-error-message.ts

```diff
@@ -0,0 +1,28 @@
+import { helper } from '@ember/component/helper';
+
+type PositionalArgs = [string];
+
+interface Signature {
+  Args: {
+    Positional: PositionalArgs;
+  };
+  Return: string;
+}
+
+export function paymentErrorMessage([failureReason]: PositionalArgs) {
```

Comment:
Known revert reasons can be found here:

https://github.com/cardstack/cardstack/blob/b19a1f838a2d95de25cdf43324a5de431c6a2d65/packages/cardpay-sdk/sdk/scheduled-payment-module.ts#L1273-L1280